libmscore/chord.cpp

```diff
@@ -861,8 +861,7 @@ void Chord::computeUp()
                   // if extrema symmetrical, average directions of intermediate notes
                   if (-ud == dd) {
                         int up = 0;
-                        int n = _notes.size();
-                        for (int i = 0; i < n; ++i) {
+                        for (int i = 0; i < _notes.size(); ++i) {
```

Comment:
Might be micro-optimization, but the old code fetched _notes.size() only once, the new code on every iteration

Reply:
solution might be to revert the order
`for (int i = _notes.size() - 1; i < 0; --i) { `